modules/ossm-about-ingress-routing-ambient-mode.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+
+// gateways/ossm-getting-traffic-into-a-mesh.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-about-ingress-routing-ambient-mode_{context}"]
+= About ingress traffic routing approaches in ambient mode
+
+[role="_abstract"]
+When using the {istio} ambient mode, you can use the {k8s} Gateway API to configure ingress traffic routing.
+
+Waypoint proxies for Layer 7 routing::
+You can deploy a waypoint proxy in the namespace that has your service to apply Layer 7 (L7) routing policies such as path-based routing or header matching. In ambient mode, waypoint proxies process L7 traffic and enforce `HTTPRoute` and `GRPCRoute` rules.
+
+[IMPORTANT]
+====
+{istio} `VirtualService` resources are not compatible with ambient mode. You must use {k8s} Gateway API resources to configure ingress traffic routing.
```

Comment:
@sridhargaddam Similar to the other comment, this statement seems to disagree with what is shown in the Istio 1.27 feature support table (Alpha). In our own verison, we called "Tech Preview" for Istio 1.27 in our own feature support table. Like the other place, can you suggest a wording that better captures the state of `VirtualService` with Istio ambient mode (It is ok to call the feature "tech preview").